.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+# Description
+<!-- Please include a summary of the change(s) and which issue is being fixed. Please provide as much detail as possible. -->
+
+
+<!-- Replace `XXX` with the concerning issue number. The "#" links this PR to its relevant issue -->
```

Comment:
Do we want to explicitly say that PRs *must* have a related issue?

Reply:
I would prefer if PRs have a related issue. If a PR doesn't have a related issue, can't they just remove the "Closes #XXX"?

Reply:
Yeah I think any PR should have a ticket associated with it

Reply:
Created #2 to enforce links to an issue